### PR TITLE
ISSUE_13

### PR DESCRIPTION
**Pull request by zoffob**
> changed events cards design
> changed events cards HTML structure (header / body / footer)
> added new animated buttons JOIN / DELETE
> event header on hover + edit on click

### CODE REVIEW

**Comment by zoffob** [comment]
Changes implemented.

File: client/src/app/components/event-card/event-card.component.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
 //card header
 //------------------------------------------
 .card-header{
-  background: #00a5db;
+  background: #123372;
```
Comment (srnec) [inline client/src/app/components/event-card/event-card.component.scss]: Just an idea... maybe it is a good time to start to define SASS variables/color map for the whole application (in some common/parent scss file)...
e.g.
$colors: (
  primary:    #3399FF,
  secondary:  #32db64,
  selecteditem-background: #054169,
  selecteditem: #fff,
  item-disabled: #ccc,
  item-disabled-background: #808080,
  icon-disabled: #ccc,
  text-disabled: #ccc,
  icon: #3c3c3b,
  text: #3c3c3b
}

and to always refer via this map to colors. In this way we will have less duplicities and more control over colors in single place.

Comment (waaana) [reply client/src/app/components/event-card/event-card.component.scss]: I made a commit, the color is not currently needed. I just added bg-primary in html. also in styles.scss I've changed few colors, like primary to erni dark blue etc..

**Comment by zoffob** [comment]
Thanks Ivan,
Pull request merged

**Pull request merged**